application/server/configuration.go

```diff
@@ -165,8 +163,6 @@ func credentialsUpdateCallback(_ string, value any) {
 	newToken, ok := value.(string)
 	if !ok {
 		msg := fmt.Sprintf("Failed to cast creds of type %T to string", value)
-		log.Error().Str("method", "storage callback token").
```

Comment:
is the log entry not needed anymore?

Reply:
Error Reporter takes care of logging. It was duplicate.